loader/loader.c

```diff
@@ -2917,7 +2917,9 @@ static VkResult read_data_files_in_search_paths(const struct loader_instance *in
     bool xdg_data_dirs_secenv_alloc = true;
 #endif
 
-#ifndef _WIN32
+#ifdef _WIN32
+    char *package_path = NULL;
+#else
```

Comment:
Nit: Could use an `#else` block here since they are mutually exclusive.

Reply:
Hm? This is an `#else` block? The pattern I'd seen through the codebase when both Windows and non-Windows had exclusive behaviors was that the `_WIN32` block was first, so I followed that pattern here. I find it confusing when the blocks take the form of `#ifndef <var>` and then `#else`, since that creates a double negative on the `#else` (if not not defined).

Reply:
Ah I can see why you'd rather have two separate blocks. I'll close this in that case.